### PR TITLE
1 prepare for cran release

### LINKED ISSUE #1: Prepare for CRAN release
Prepare for CRAN release

- [x] Update CRAN comments
- [x] Update release type
- [x] Update News
- [x] Run external checks across platforms
- [x] Update load and attach message
- [x] Change all instances of 'ccdR' to 'ctxR'

### PR DESCRIPTION
**Pull request by rachmaninoffpk**
This closes #1 .

### CODE REVIEW

File: R/register_ctx.R [hunk 1]
```diff
@@ -54,72 +54,72 @@
...
 #' @examplesIf has_ctx_key() & is.na(ctx_key() == 'FAKE_KEY')
 #' # Register key for this session
-#' register_ccdr(key = 'YOUR API KEY')
+#' register_ctxR(key = 'YOUR API KEY')
 #' # Register key over sessions
-#' register_ccdr(key = 'YOUR API KEY', write = TRUE)
+#' register_ctxR(key = 'YOUR API KEY', write = TRUE)
 
-register_ccdr <- function(key, write = FALSE) {
+register_ctxR <- function(key, write = FALSE) {
```
Comment (madison-feshuk) [inline R/register_ctx.R:114]: Perhaps we should rename this function as "register_ctx_api_key" ? Or could model after tcpl like "ctxR_conf" (for "Configure")

Update line 1 as "Register CTX API Key for ctxR" or "Configure environment with CTX API key"

**Review by madison-feshuk — COMMENTED** [review]
ctxR works for me! Thoughts on renaming "register_ctxR" function? @rachmaninoffpk @brown-jason

File: R/register_ctx.R (hunk 1 above)
Comment (rachmaninoffpk) [reply R/register_ctx.R:114]: I would opt for the "register_ctx_api_key" and "Register CTX API Key for ctxR" pair.

**Review by brown-jason — APPROVED** [review]
── R CMD check results ─────────────────────────────────────────────────────────────────────── ctxR 1.0.0 ────
Duration: 1m 5.1s

0 errors ✔ | 0 warnings ✔ | 0 notes ✔

**Pull request merged**